style.css

```diff
@@ -98,3 +98,1104 @@ body {
   margin-top: 3.2rem;
   background-position: center;
 }
+
+.head-projects {
+  height: 5rem;
+  width: 18.1rem;
+  left: 1.6rem;
+  top: 67.2rem;
+  font-family: Playfair Display, sans-serif;
+  font-size: 5rem;
+  font-weight: 400;
+  line-height: 5rem;
+  text-align: left;
+  position: absolute;
+  text-transform: capitalize;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.card-container {
+  display: grid;
+  grid-template-columns: auto;
+}
```

Comment:
- [ ] Please make sure your design must be same as [Figma's design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9&t=y6XOaeJGzPd4Vwqo-0) but in your case, your design is not the same as Figma's design because this is a requirement if you check your design I think you missed `justify-items: center` so kindly use this property. This property will solve your issue so please read the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) carefully. Also, check below :point_down:  my suggestion and see comparison screenshots :slightly_smiling_face: 

```suggestion
.card-container {
  display: grid;
  grid-template-columns: auto;
  justify-items: center;
}
```

**Your Design:**
![screencapture-127-0-0-1-5500-2023-02-24-14_29_05 (1) (1)](https://user-images.githubusercontent.com/45226848/221153622-a9715623-310e-4541-bacb-96102ddd3edc.png)

**Figma Design:**
![Screenshot from 2023-02-24 15-19-01](https://user-images.githubusercontent.com/45226848/221154015-d214ead2-9c01-4a10-b858-6ad100bb9b7a.png)